enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -1862,7 +1862,7 @@ func (c *FirecrackerContainer) create(ctx context.Context) error {
 	if c.vmConfig.InitDockerd {
 		dockerDaemonConfig, err := getDockerDaemonConfig()
 		if err != nil {
-			return status.InternalErrorf("Could not write Docker daemon config: %s", err)
```

Comment:
Did this cause a problem?

Reply:
no, just making sure we always return retryable errors from this function. I think that partly contributed to the issue this morning